### ISSUE TITLE
Release current app to the app store

### ISSUE THREAD

**Issue by tordans** [bryceco/GoMap#190]
The last public release was `Sep 20, 2018`.
https://itunes.apple.com/us/app/go-map/id592990211?mt=8

It would be awesome, to have the latest version available.

Are there thinks that block a release? If so, I am happy to help where I can.
Maybe we can use this ticket to track the open issues?

ps: I noticed today, that https://github.com/bryceco/GoMap/issues/135 is still an issue for the version I use. And there are other very helpful improvements that I would love to use :).

**Comment by bryceco** [comment]
I'm going to be traveling until the end of the month. I'll push out a new release when I'm back.

**Comment by tordans** [comment]
Perfect, @bryceco.

Is there anything we can do to help until then?
AFAIK its not on the Preflight(?) app ATM, so we cannot do beta testing for now. Can you start that now or will we start it at the end of the month?

Also, any other regular deploy tasks?
Like reviewing the presets from iD for merging?
Is https://github.com/bryceco/GoMap/pull/170 still on the release train?

Thanks

**Comment by bryceco** [comment]
There is a certain amount of busy work with syncing to the latest ID presets. ID typically adds presets that require custom coding, so we have to try out the latest presets and then fix the issues that ID wrote code to fix (usually crash inducing). 

Then we we need to import the translations for the latest presets. Unfortunately transiflex has their files in yml, but iOS doesn’t have a yml parser so we have to manually translate them to xml (lots of websites do this easily for you). Not too difficult but not totally straight forward.

**Comment by wtimme** [comment]
> a certain amount of busy work with syncing to the latest ID presets

> we have to manually translate them to xml

Can you go into detail what needs to be done? I'd be happy to help and automate as much as possible here. Do you have experience with Fastlane?

@tordans, do you have access to a machine running macOS? You could deploy a debug version to your iOS device. I could add some instructions to the README, if needed.

**Comment by tordans** [comment]
> do you have access to a machine running macOS? You could deploy a debug version to your iOS device. I could add some instructions to the README, if needed.

Yes, I do run macOS.  Hovewever I am also on the testflight-list and already using the new test flight that bryceco started on Friday. Since that reaches more people, I think its the better approach.

Btw, https://github.com/bryceco/GoMap/issues/206 seems to be an issue for the release. I added it to the list above.

**Comment by wtimme** [comment]
Hey @bryceco, is there anything I can do to help publish the next release, e. g. write a changelog, set up automation, fix major bugs, etc.?

**Comment by bryceco** [comment]
We should both make a pass through the issue tracker and flag anything we feel is a blocking issue. I also need to go through a bunch of email requests and I’ll add issues for those that are most critical. Then we’ll release a final beta, wait a week for testing, then release. Sound good?

**Comment by bryceco** [comment]
I’ve added a Priority label to items I’d like to fix before submitting to App Store

**Comment by wtimme** [comment]
Thanks for your response!

Personally, I prefer smaller, more frequent releases over large ones, and I feel that 1.7.0 is going to be quite large. So I suggest that we fix all _critical_ critical bugs that **necessarily** need to be fixed right now (and are worth blocking the release) and then release a new version immediately, without waiting the other issues.

If we look at the development time that we have as 100%, I would rather spend 10% on blockers/fixes now and then have 80% left for fixes _after_ the release than the other way around.

To make my point a bit clearer, let's have a look at the issues with the label "Priority".

- Place duplicated pin below the yellow cross (#134)
  - Sounds like an optimization rather than a bug. @tordans also mentions that in his first commt: "The improved behavior (...)".
  - **Suggestion: Skip it for 1.7.0**
- Reorganize user feedback (#296)
  - Yep, I agree that the section could use a bit of re-organization, but it is nothing that blocks the user from mapping.
  - **Suggestion: Skip it for 1.7.0**
- Flickering of panel after selecting preset (#282)
  - Doesn't look too nice, yes, but it doesn't block users from mapping. (@tordans, please correct me if I'm wrong here.)
  - **Suggestion: Skip it for 1.7.0**
- Thank the code-contributors (#211)
  - Nice idea, but not something that we _need_ to ship right now.
  - Plus, we are not even 100% sure what exactly we want to do there. Let's clarify the questions and our ideas first, and take some time for it.
  - **Suggestion: Skip it for 1.7.0**
- No autocomplete for number values (#166)
  - Annoying, yes, but it's been like that for apparently more than a year, so it doesn't seem to block users from mapping. It's merely inconvenient.
  - **Suggestion: Skip it for 1.7.0**
- 0 (#154)
  - Again, this bug seems to have been in the app since more than a year.
  - @tordans wrote: "But personally I don’t think it’s a real problem." So I think we can fix it later, and release it with a later version.
  - **Suggestion: Skip it for 1.7.0**
- Navigating relations (#144)
  - This looks like a feature rather than an important bug report. For sure, definitely something we can implement, but it's not urgent for now.
  - **Suggestion: Skip it for 1.7.0**

Personally, the only issue that I think we should probably fix/resolve is #310, since this was introduced with our latest changes to the icons.

So, to sum it up: I suggest we fix #310 and then release a new version. The other issues can wait until a later version.

**Comment by bryceco** [comment]
#282 is a regression and I definitely want it fixed before release. If we fix it and #310 I’m okay with pushing a release before proceeding with the others.

**Comment by wtimme** [comment]
Now that both issues are closed, we can go ahead here. 🎉

@bryceco, let me know if you need a helping hand.

**Comment by bryceco** [comment]
1.7.0 is now released
